README.md

```diff
@@ -10,15 +10,15 @@ This is a new project and is in alpha state.
 
 ## 1. Deployment
 
-Prerequisite a kubernetes cluster e.g. using ACS engine or AKS
+Prerequisite a kubernetes cluster e.g. using ACS engine or AKS.
```

Comment:
s/Prerequisite a kubernetes cluster.+/Access to a Kubernetes cluster with "cluster-admin" permission is the only prerequisite./